pages/secure.py

```diff
@@ -0,0 +1,23 @@
+from playwright.sync_api import Page
+
+class SecurePage:
+    def __init__(self, page: Page):
+        self.selectors = _Selectors()
```

Comment:
Put selectors under page and url assignments to keep consistent with the existing page objects (same for form_auth page object)

Reply:
Done